File: src/CSharpLanguageReference/Properties.cs
using System;
using System.Collections.Generic;

namespace CSharpLanguageReference
{
    public class Properties
    {
        // Basic syntaxes
        public int Property1 { get; set; }
        public MyType Property2 { get; set; }
        public MyType.MySubType Property3 { get; set; }
        public List<string> Property4 { get; set; }
        public string[] Property5 { get; set; }
        public object Property6 { get; set; }
        public ISomething Property7 { get; set; }
        public MyType.MyGenericType<int> Property8 { get; set; }

        // nullables
        public int? Property11 { get; set; }
        public MyType.MySubStruct? Property12 { get; set; }
        public Nullable<int> Property13 { get; set; }
        public Nullable<MyType.MySubStruct> Property14 { get; set; }
        public object? Property15 { get; set; }
        public ISomething? Property16 { get; set; }
        public Nullable<object> Property17 { get; set; }
        public Nullable<ISomething> Property18 { get; set; }
        public Nullable<MyType.MyGenericType<int>> Property19 { get; set; }


        // Generic variations
        public List<Nullable<int>> Property21 { get; set; }
        public List<MyType> Property22 { get; set; }
        public List<MyType.MySubType> Property23 { get; set; }
        public List<int?> Property24 { get; set; }
        public List<string[]> Property25 { get; set; }
        public List<object> Property26 { get; set; }
        public List<ISomething> Property27 { get; set; }
        public List<List<string>> Property28 { get; set; }
        public List<MyType.MyGenericType<int>> Property29 { get; set; }
    }
}

File: VisualBasicLegRef/VisualBasicLangRef/Properties.vb
Class Properties
    Public Property Property1 As Integer
    Public Property Property2 As String
    Public Property Property3 As MyType
    Public Property Property4 As MyType.MySubType
    Public Property Property5 As List(Of String)
    Public Property Property6 As Object
    Public Property Property7 As ISomething
    Public Property Property8() As String
    Public Property Property9 As MyType.MyGenericType(Of Integer)

    Public Property Property11 As Integer?
    Public Property Property12 As List(Of Integer?)
    Public Property Property13() As Integer?
    Public Property Property14 As MyType.MyGenericType(Of Integer?)
    Public Property Property15 As Nullable(Of Integer)
    Public Property Property16 As List(Of Nullable(Of Integer))
    Public Property Property17() As Nullable(Of Integer)
    Public Property Property18 As MyType.MyGenericType(Of Nullable(Of Integer))

    Public Property Property21 As List(Of MyType)
    Public Property Property22 As List(Of MyType.MySubType)
    Public Property Property23 As List(Of MyType.MyGenericType(Of Integer))
    Public Property Property24 As List(Of List(Of String))
    Public Property Property25 As List(Of ISomething)



    Public Property Property41 As (Integer, String)
    Public Property Property42 As (id As Integer, name As String)

End Class
